ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/README.md

```diff
@@ -0,0 +1,66 @@
+# Synthea Automation Scripts and Docker Image
+
+This directory contains several Synthea-related scripts pertaining to generating synthetic data
+using Synthea, as well as the definition for a Docker image (`bfd-mgmt-synthea-generation`) that
+includes said scripts.
+
+## Generating Synthea Data Locally Using the `bfd-mgmt-synthea-generation` Docker Image
+
+It is _preferred_ to generate Synthea data locally using the `bfd-mgmt-synthea-generation` Docker
+image, as it has the following advantages over using the `run_synthea_locally.sh` script:
+
+1. The Docker image comes pre-built with the necessary mapping files, the Synthea executable, and
```

Comment:
so as far as Synthea executable, how will versioning happen; I belive they actually update their release jar fairly often (i.e., maybe 1 or 2x / month). So does that imply we need to re-build docker image on some timely basis?

Reply:
Yep, that's correct. Fortunately it's a fairly quick process -- from the few builds I've done using the `bfd-mgmt-synthea-generation-build` Jenkins pipeline it should only take a few minutes to update the image whenever a new release occurs.